initrd/bin/seal-hotpkey

```diff
@@ -70,20 +82,63 @@ else
 	HOTPKEY_BRANDING="HOTP USB Security Dongle"
 fi
 
-echo -e ""
-read -s -p "Enter your $HOTPKEY_BRANDING Admin PIN: " admin_pin
-echo -e "\n"
+# Truncate the secret if it is longer than the maximum HOTP secret
+truncate_max_bytes 20 "$HOTP_SECRET"
```

Comment:
Why are we still truncating secret?

Reply:
@tlaurion See comments above, will tag you again there